components/CopyValueToClipboard.tsx

```diff
@@ -13,12 +13,12 @@ interface CopyValueToClipboardProps extends CopyToClipboardButtonProps {
 
 const CopyValueToClipboard: FC<CopyValueToClipboardProps> = ({
   label,
-  labelProps = {},
+  labelProps = { as: 'h4' },
```

Comment:
issue(non-blocking): This is not a blocking issue because I see that the current behavior is to use an <h4> for this tag, but IMO we shouldn't be defaulting to "h4" here.

Headings should be in order (e.g. an <h1> should be followed by <h2> then <h3>, and so on), so tying a heading level to a generic component like this feels off.

For this, I would probably wrap the `{label}` in a `<Text>` instead of `<Box>`, which I believe Chakra will use a <p> to render by default, and let the call site override that as you're doing here.

All that said, though, since the current behavior is to use `as="h4"`, this is not a blocking issue.